pkg/policyprovider/configpolicy/configpolicy.go

```diff
@@ -74,3 +104,43 @@ func (enforcer PolicyEnforcer) ErrorToVerifyResult(ctx context.Context, subjectR
 	reports = append(reports, errorReport)
 	return types.VerifyResult{IsSuccess: false, VerifierReports: reports}
 }
+
+// OverallVerifyResult determines the final outcome of verification that is constructed using the results from
+// individual verifications
+func (enforcer PolicyEnforcer) OverallVerifyResult(ctx context.Context, verifierReports []interface{}) bool {
+	// use boolean map to track if each artifact type policy constraint is satisfied
+	verifySuccess := map[string]bool{}
+	for artifactType := range enforcer.ArtifactTypePolicies {
+		// add all policies except for default
+		if artifactType != defaultPolicyName {
+			verifySuccess[artifactType] = false
+		}
+	}
+
+	for _, report := range verifierReports {
+		castedReport := report.(verifier.VerifierResult)
+		// the artifact type of the verified artifact matches type specified in policy
```

Comment:
If all verifications for a type that is not set in the config (with default policy)  fails, what is the overall result?

Reply:
Currently OverallVerifyResult only considers the artifact types that are explicitly specified. And those are the only ones that are taken into account for overall success. So it disregards the success/failure of an unspecified artifact type.